src/Okta.Sdk.UnitTests/Internal/TestDefaultRequestExecutor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Okta.Sdk.Configuration;
+using Okta.Sdk.Internal;
+
+namespace Okta.Sdk.UnitTests.Internal
+{
+    public class TestDefaultRequestExecutor : DefaultRequestExecutor
```

Comment:
Shouldn't we implement the interface instead?

Reply:
See previous comment.